### ISSUE TITLE
error while loading shared libraries: libjsoncpp.so.24

### ISSUE THREAD

**Issue by aleister888** [mihirlad55/polybar-dwm-module#34]
polybar: error while loading shared libraries: libjsoncpp.so.24: cannot open shared object file: No such file or directory

using jsoncpp 1.9.5-1 and latest AUR polybar-dwm-module-package, downgrading jsoncpp 1.9.4 solves the issue, it seems https://github.com/polybar/polybar/issues/2051#issuecomment-610364650 this is the problem, but recompiling using yay --answerclean/--rebuildall doesnt solve it

**Comment by pushpraj-rmx** [comment]
Facing same error. please resolve

**Comment by pushpraj-rmx** [comment]
> polybar: error while loading shared libraries: libjsoncpp.so.24: cannot open shared object file: No such file or directory
facing same error
> using jsoncpp 1.9.5-1 and latest AUR polybar-dwm-module-package, downgrading jsoncpp 1.9.4 solves the issue, it seems [polybar#2051 (comment)](https://github.com/polybar/polybar/issues/2051#issuecomment-610364650) this is the problem, but recompiling using yay --answerclean/--rebuildall doesnt solve it

did you find any solution ?

**Comment by pushpraj-rmx** [comment]
Delete the the directory '~/.cache/yay/polybar-dwm-module'
rebuild polybar using yay -Su polybar-dwm-module 
It solves the issue, and there is no need to downgrade any package. Just rebuild polybar.

If it solves you issue, please let  me know and mark as solved.

**Comment by aleister888** [comment]
no, it doesn't, --answerclean deletes cache already, nvm manually deleting doesnt solve the issue neither
